File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) 
{
    int height;
    do {
        printf("Height: ");
        height = get_int();
    } while (height > 23 || height < 0);
    
    for (int i = 1; i <= height; i++) //which row
    {
        for (int j = height -i; j > 0; j--) //spaces-left
        {
            printf(" ");
        }
        for (int h = 1; h <= i; h++) // blocks # -left
        {
            printf("#");
        }
        printf("  "); // spaces center
        for (int h = 1; h <= i; h++) // blocks # -right
        {
            (h == i) ? printf("#\n") : printf("#");
        }
    }
}

File: pset6/mario.py

height = 0

while True:
    print("Height: ", end="")
    height = int(input())
    if height > 0 and height < 24:
        break

for i in range(1, height+1): # print "i"th row
    for j in range(height-i, 0, -1): # Spaces on the left
        print(" ", end="")
    
    for j in range(1, i+1): # Blocks on the left
        print("#", end="")
        
    print ("  ", end="") # Spaces at the center
    
    for j in range(1, i+1):
        print("#") if j == i else print("#", end="") # Blocks on the right side
